services/bots/src/github-webhook/handlers/validate-cla.ts

```diff
@@ -85,7 +87,7 @@ export class ValidateCla extends BaseWebhookHandler {
     }
 
     const commits = await this.getCommits(eventData);
-    await this.checkAuthors(eventData, commits.data);
+    await this.checkAuthors(params, eventData, commits.data);
```

Comment:
Can't we add a class to store pending data to `params` instead of using global variables?